### ISSUE TITLE
Question: How is the cheatsheet generated?

### ISSUE THREAD

**Issue by Sighery** [ranger/ranger#1712]
This isn't an issue per se, but a question.

I was searching around to see if there's any information on how the `cheatsheet.svg` was generated/maintained, but I couldn't find any information. It also doesn't seem to be included into any of the build `make` scripts so I assume it's something custom.

I'd like to add a few other default keybindings to it, like `Insert` for `touch`. If it's some automated process, I'd be okay with making a PR to both document it and a `make` case for it. And if useful, another PR to add into the current cheatsheet the `Insert` keybinding, and maybe other default ones I might find useful that aren't yet covered.

**Comment by toonn** [comment]
It's completely manual afaik. Feel free to simply edit it in inkscape or to automate generation. Both would be welcome contributions : )

**Comment by Sighery** [comment]
I did a little looking around and I wasn't able to find any good tools for cheatsheets generation. Might just be my Google-fu skills failing me, though.

The few things I was able to find were websites and such, which are closed source, but are also not meant for automation, so I don't even count them. Nothing like a tool that when given a list of keymaps and their values will automatically generate a cheatsheet off them.

If you (or anyone reading this) knows of a suitable tool feel free to let me know and I'll take a look and see if I can make it work.

For now I'll just manually edit it. Off to figure out Inkscape now :laughing:
